### ISSUE TITLE
App icons Unassigned approval requests

### ISSUE THREAD

**Issue by LMapes** [carbon-design-system/design-language-website#759]
Investigate UX possibilities for users interested in a particular Unassigned app icon 
- icon that sends them to a request form instead of performing a download?
- grey out the Unassigned icon while approval is underway?
- separate Unassigned icons into separate category in Library tab with text explaining approval icon action?

FYI @kennethhartmann

**Comment by DianaStanciulescu** [comment]
Some thoughts: 
Unassigned icons are used in 2 situations:
1. A product team sees an unassigned icon they want to use for their app, they request to use it. The approval should be pretty straightforward and quick.
2. A designer suggests one ore more of the unassigned icons to a product team, as well as maybe new designs; until the end of the process, nobody knows which icon will be chosen.

For situation 1, I am thinking of 3 possibilities, depending also on what is technically possible:

1A: the unassigned icon in the library can have a request button on hover. if clicked the button should: opn a github issue in the right repo with a link to the clicked on icon and predefined text requesting approval. Once the issue is submitted, the icons in the gallery can have the text changed to "request in progress". this requires a lotof things to be connected, so not sure if it is possible.

1B: the unassigned icon can have a "request" option on hover that leads to the approvals section in the production tab. The text there should include the process for requesting unassigned icons. the process of opening an issue is then done manually by the requester; maybe the issue can have a tag that can prompt somebody to then also manually change the status of the icon in the gallery; however, this may take too long and before this is accomplished the icons may be already assigned and approved. 

1C the process for requesting an unassigned icon is only described in the production tab, under approvals; there is no status update in the library; the process explains what happens when there is a conflict (eg: the issue opened first wins)

Case no2:
A designer may suggest many unassigned icons; I do not think it is worth blocking all of them since in many cases they are not chosen; plus, this change of status in the gallery being manual there will be A LOT of work for somebody to keep track of all of this.

Conclusion and my personal opinion:
In Production / Approvals, thee should be text explaining the process for using unassigned icons, and what happens when ther is a conflict; maybe this is even explained right at the top of the production page.  Other than this, unless this can be completele automatized, I would not change the status of the icons, the process would take too long and cause more confusion than help; a one click change of status would be nice if this can be implemented (a sort of admin possibility of changing the status of an icon directly from the site).

@LMapes @peter-garvin @mjabbink

**Comment by mjabbink** [comment]
Or, we remove the unassigned from the library. We can utilize these in the design process as always and suggest these even if a team has design proposals.

**Comment by DianaStanciulescu** [comment]
That can work if we make a separate repo with these so we know where to take them from; I was worried the fill and plex categories will suffer since they have fewer icons but it can still work

**Comment by LMapes** [comment]
@mjabbink @DianaStanciulescu @peter-garvin 
After an exchange in ibm-brand yesterday where someone saw an Unassigned icon they wanted for their product, what do you think of the possibility of adding an arrow —> icon (instead of download as we do for Pictos) that pops up a tooltip saying 'Claim this icon'. 

It could pop them over to a paragraph describing process on the Production page, or go directly to a new '[app icon claim]' story template in GH. That way, the team could gather all the essential data from the claiming BU:
Product owner 
BU/PO contact info
legal approved product name (or suggested if approval is underway)
screenshot of icon they want (or some other tech cool way of adding this to the GH issue)
etc. 
This story could also include caveats from our end:
Expect acknowledgement in 24 hours
Approval response in 7-10 business days or whatever fits

**Comment by mjabbink** [comment]
@DianaStanciulescu Should we close this?

**Comment by DianaStanciulescu** [comment]
I don't think we solved it but nobody is making any decisions on this process and things seem to be working ok with out current process, so probably yes. @peter-garvin  what do you think?

**Comment by mjabbink** [comment]
Either we close or move into the “Brand priority” column. It can’t site in N/A column

**Comment by DianaStanciulescu** [comment]
I would close in this case. probably not a brand priority.

**Comment by DianaStanciulescu** [comment]
this sounds like an awesome idea!

**Comment by mjabbink** [comment]
@peter-garvin @DianaStanciulescu agree!
